### PR TITLE
Relocate the dropdown filter

### PR DESCRIPTION
**Pull request by aanish27**
Resolved please review @Raza9798 #87

### CODE REVIEW

File: resources/views/company/index.blade.php [hunk 1]
```diff
@@ -108,7 +108,7 @@
               </div>
             </div>
            <table id="myTable" class="table table-hover table-nowrap table-bordered shadow-sm"  width="100%" ></table>
-            <select id="filter-country" class="form-select form-control form-control-sm py-1 px-1 d-none" style="width: 200px" aria-label="Default select example">
+            <select id="filter-country" class="form-select form-control form-control-sm py-0 px-1 d-none" style="width: 150px" aria-label="Default select example">
```
Comment (Raza9798) [inline resources/views/company/index.blade.php:111]: use bootstrap class instead of hardcoding inline css

**Review by Raza9798 — CHANGES_REQUESTED** [review]
changes requested

File: resources/views/admin.blade.php [hunk 2]
```diff
@@ -144,7 +144,7 @@
 
                 <table id="myTable" class="table table-hover table-nowrap table-bordered shadow-sm"  width="100%"></table>
 
-                <select id="filter-position" class="form-select form-control form-control-sm py-1 px-1 d-none" style="width: 200px" aria-label="Default select example">
+                <select id="filter-position" class="form-select form-control form-control-sm py-0 px-1 d-none" style="width: 150px" aria-label="Default select example">
```
[the remark this answers is not in the document]
Comment (aanish27) [reply resources/views/admin.blade.php]: Resolved

File: resources/views/company/index.blade.php (hunk 1 above)
Comment (aanish27) [reply resources/views/company/index.blade.php]: Resolved

**Comment by aanish27** [comment]
> changes requested

Resolved please review again @Raza9798 #87

**Review by Raza9798 — APPROVED** [review]

**Pull request merged**